opentsdb/opentsdb.go

```diff
@@ -62,10 +70,25 @@ func (s *Server) ListenAndServe(listenAddress string) {
 		return
 	}
 
+	// Set up the background HTTP server that we
+	// will pass http request to via a channel
+	mux := http.NewServeMux()
+	mux.HandleFunc("/api/metadata/put", func(w http.ResponseWriter, r *http.Request) {
```

Comment:
This is an extension provided by bosun I think, but it accessed by scollector. Adding this prevents an error showing the scollector logs.
/api/put is the TSDB data upload endpoint. This patch, combined with the basic telnet stuff already supported, mean influx can support tcollector and scollector, which are the two main TSDB data collection agents.